README.md

```diff
@@ -115,37 +115,39 @@ public class HelloWorld {
 ```
 
 ----------
-Conditional module loading
+Conditional bindings
 ----------------------------------
-Karyon supports conditional module loading to auto install Guice modules based on the application runtime environment.  Conditionals can depend on property values, modules having been installed, bindings, etc.  An example use case would be to set the appropriate bindings for running Eureka locally as opposed to running in the cloud without requiring the developer to know which specific bindings to override.  For conditional bindings to work all the jars must be in the classpath and the modules made known to Karyonvia a ModuleListProvider (such as ClassPathModuleListProvider, ServiceLoaderModuleListProvider, etc...).  By default karyon will include any modules under the 'com.netflix.karyon' package.
+Applications and libraries frequently need different bindings based on the context in which the application is running.
+Conditional bindings may be specified in a Guice module using @ProvidesConditionally.  This features is syntactic sugar
+instead of having to write a complex Provider, put conditional logic in a Guice module or write complex conditional code
+when determining which Guice modules to install. 
 
 ```java
 Karyon.create()
-      .addModuleListProvider(ModuleListProvides.forPackage("org.example")
+      .addModules(new AbstractModule() {
+           @Override
+           protected void configure() {
+           }
+           
+           @ProvidesConditionally(isDefault=true)
+           @ConditionalOnProfile("local")
+           protected Foo getFooWhenRunningLocally() {
+                return new FooForLocalDevelopment();
+           }
+           
+           @ProvidesConditionally
+           @ConditionalOnEc2
```

Comment:
We can't determine this at compile time because conditionals are inherently runtime concepts.  We could however determine at injector creation if there is a conflict and fail early on.